ocs_ci/utility/aws.py

```diff
@@ -427,6 +460,54 @@ def get_ec2_instance_volumes(self, instance_id):
         volumes = instance.volumes.all()
         return [vol for vol in volumes]
 
+    def create_security_group(self, group_name, dict_permissions, vpc_id):
+        """
+        Create security group with predefined group name and permissions
+
+        Args:
+            group_name (str): Group name (aws tag: "Group Name")
+            dict_permissions(dict): The security group's inbound/outbound permissions
```

Comment:
dict_permissions (dict)
vpc_id (str)

--> space between 'param_name' and (type)